model/Aktivita/Aktivita.php

```diff
@@ -3382,7 +3382,7 @@ public static function zProgramu(
                 1 => StavAktivity::NOVA,
                 2 => TypAktivity::interniTypy(),
             ],
-            order: 'ORDER BY DAY(zacatek), ' . dbQi($razeni) . ', HOUR(zacatek), nazev_akce',
+            order: 'ORDER BY DAY(zacatek) - IF(HOUR(zacatek) > ' . dbQv(PROGRAM_ZACATEK) . ', 0, 1), ' . dbQi($razeni) . ', DAY(zacatek), HOUR(zacatek), nazev_akce',
```

Comment:
Je OK, že řadíme nejdříve podle upraveného dne ± 1, potom řadíme podle **neupraveného** dne a až potom podle hodin? Nechceme řazení podle neupraveného dne úplně vyhodit?

Reply:
To je to, co jsem ti psal - pokud je jedna aktivita v sobotu večer a druhá v sobotu v noci (tj. v neděli ve 2 ráno např.), tak bez toho neupraveného dne se takto zařadí ta noční aktivita přes tu denní (což nedává žádný smysl a vykreslí se to blbě).